test/mix_unused/analyzers/unreachable/usages/absinthe_discovery_test.exs

```diff
@@ -0,0 +1,93 @@
+defmodule MixUnused.Analyzers.Unreachable.Usages.AbsintheDiscoveryTest do
+  use ExUnit.Case
+
+  alias MixUnused.Analyzers.Unreachable.Usages.Context
+  alias MixUnused.Analyzers.Unreachable.Usages.AbsintheDiscovery
+  alias MixUnused.Meta
+
+  import Mock
+
+  test "it discovers generated function" do
+    with_mock File,
```

Comment:
I am not really fan of that mocking. It somehow feels that the code is doing a little bit too much.

Reply:
The alternative is to store additional files only used by tests, but in this case mocking `File.read!` seems to be a good compromise: the sources are really close to the related tests and the behaviour of each test is obvious.